nodewatcher/modules/monitor/http/survey/cgm.py

```diff
@@ -0,0 +1,10 @@
+from nodewatcher.core.generator.cgm import base as cgm_base
+
+
+@cgm_base.register_platform_module('openwrt', 50)
+def clients(node, cfg):
+    """
+    Register packages needed for reporting the clients.
+    """
+
+    cfg.packages.update(['nodewatcher-agent-mod-clients'])
```

Comment:
Is this necessary for my module? I copied it from clients module but I don't think I'm using it.